### DISCUSSION TITLE
Merging strategy for Universal Calculator development

### DISCUSSION THREAD

**Discussion by Satish-Mutyala** [Satish-Mutyala/T24S2T1EXTG12#6] [General]
Hi Team, could you please let everyone know once you merged your feature branch into development branch?

**Comment by WeiZhou-dev** [comment]
Sure, I am working on my feature branch now and will let you know once I merge it into the development branch.

**Comment by JeongseopYun** [comment]
Hello Satish, I have merged my feature to the development branch. Thank you.

**Reply by Satish-Mutyala** [reply]
Thanks @JeongseopYun 

I am going to merge my feature branch into development now.

**Reply by Satish-Mutyala** [reply]
I have merged feature_miscellaneous_converter into development branch

**Reply by WeiZhou-dev** [reply]
I have merged my mortgage feature branch  as well
